src/browser/components/icons/Icons.tsx

```diff
@@ -220,10 +220,10 @@ export const AboutIcon = ({ isOpen, title }: SidebarIconProps): JSX.Element => (
 )
 
 export const SlidePreviousIcon = (): JSX.Element => (
-  <IconContainer icon={arrowLeft1} width={20} />
+  <IconContainer icon={arrowLeft} width={20} />
```

Comment:
Can we make these widths constants?

```
const LARGE_SIZE = 20
const SMALL_SIZE = 12
```

Reply:
Sure! There's another one as well, adding that one as well